rele/client.py

```diff
@@ -14,6 +14,7 @@
 
 
 class Subscriber:
+    DEFAULT_ACK_DEADLINE = 60
```

Comment:
How about evaluating this conditional
```suggestion
    DEFAULT_ACK_DEADLINE = int(os.environ.get('DEFAULT_ACK_DEADLINE')) or 60
```
?

Reply:
I would like to suggest to use the default 10 seconds as ack deadline

Reply:
@andrewgy8 We thought about your suggestion at first place when implementing the feature. However, we wanted to go a step further and evaluate the `DEFAULT_ACK_DEADLINE` variable at the moment we actually use it. We thought this could add more flexibility to the settings and ease the unit testing as well. What do you think? Maybe we missed the point here.

About the default value. Since we complained a bit about the default ack deadline at the moment we discussed this feature, we opted to give our own default value. This is greater than the one Google sets and we also think it can be healthy to control our default value in order to provide more predictability (we don't know whether Google decides to change that default single-handedly at a certain moment).

Reply:
A good library also has reasonable defaults, i don't see a problem with using a bigger value than Google

Reply:
I agree that 60 seconds is fine.